draft-ietf-tls-subcerts.md

```diff
@@ -461,7 +461,7 @@ CertificateEntry of its end-entity certificate; the server SHOULD ignore
 delegated credentials sent as extensions to any other certificate.
 
 The algorithm field MUST be of a type advertised by the server
-in the "signature_algorithms" extension of the CertificateRequest message
+in the "signature_algorithms" extension of the CertificateRequest message,
 and the dc_cert_verify_algorithm field MUST be of a type
 advertised by the server in the SignatureSchemeList
 and is considered not valid otherwise.  Servers that receive non-valid
```

Comment:
```suggestion
in the "signature_algorithms" extension of the CertificateRequest message,
and the dc_cert_verify_algorithm field MUST be of a type
advertised by the server in the SignatureSchemeList; otherwise, the credential is
considered not valid.  Servers that receive non-valid
```